codeship_yaml/main.py

```diff
@@ -49,17 +50,44 @@ def exec_section(self, section_name):
             commands = []
 
         for cmd in commands:
+            print(color_message('========== Running: {0}'.format(cmd), 'blue'))
+            start = time.time()
             status = os.system(cmd)
+            duration = round(time.time() - start, 2)
+
+            dur_msg = "{1}s: {0}".format(cmd, duration)
+
             if status > 0:
-                fail(exit_code=status)
+                finish_msg = "========== Errored after {0}".format(dur_msg)
+                fail(message=finish_msg, exit_code=status)
+            else:
+                finish_msg = "========== Finished in {0}".format(dur_msg)
```

Comment:
Please use simple quotes (`'`) for all simple strings, unless double quotes (`"`) make sense for readability / code beauty, and use empty braces (`{}`) or named string arguments.